lib/v2/screens/authentication/recover/recover_account_found/interactor/recover_account_found_bloc.dart

```diff
@@ -34,14 +40,15 @@ class RecoverAccountFoundBloc extends Bloc<RecoverAccountFoundEvent, RecoverAcco
       RecoverGuardianInitialDTO result = await FetchRecoverGuardianInitialDataUseCase().run(state.userGuardians);
       yield FetchRecoverRecoveryStateMapper().mapResultToState(state, result);
       yield* _mapStartTimerToState();
-    }
-    else if (event is Tick) {
+    } else if (event is Tick) {
       if (event.timer > DateTime.now().millisecondsSinceEpoch ~/ 1000) {
         yield RemainingTimeStateMapper().mapResultToState(state, event.timer);
       } else {
         await _tickerSubscription?.cancel();
         // TODO(gguij002): Show account ready to claim UI
       }
+    } else if (event is OnClaimAccountTap) {
+      _authenticationBloc.add(OnImportAccount(account: state.userAccount, privateKey: settingsStorage.privateKey!));
```

Comment:
I always thought that we could avoid bringing the Authentication bloc as a dependency to another. 
If the event create a PageCommand and then the Authentication bloc event is handled from a listener in the screen.

It is the same but the DI would be avoided.

Reply:
i see, yeah, also possible.